lib/invoker/power/setup/distro/ubuntu.rb

```diff
@@ -4,6 +4,9 @@ module Invoker
   module Power
     module Distro
       class Ubuntu < Debian
+        def restart_services
+          system('service dnsmasq restart')
+        end
```

Comment:
Why TLD change includes this stuff? This is old upstart way of starting services. All new Ubuntu distros ship with systemd and I thought we moved to systemd for this stuff.